chips/nrf52/src/comp.rs

```diff
@@ -0,0 +1,303 @@
+use kernel::common::cells::OptionalCell;
```

Comment:
Need `//!` comment.

Reply:
Fixed.

Reply:
Please use the name `acomp` as the crate name to distinguish this from a regular (digital) comparator, as well as be in sync with the other names (e.g., AcChannel). As Brad notes, `comp` is confusing, let's not follow their lead and blame them.

Reply:
Fixed. I didn't change the interrupt name in peripheral_interrupts though (I wasn't sure if I should change it or not).